autoload/sy/util.vim

```diff
@@ -24,6 +24,10 @@ endfunction
 
 " #refresh_windows {{{1
 function! sy#util#refresh_windows() abort
+  " return if signify is not active
+  if empty(getbufvar(bufnr(''), 'sy'))
```

Comment:
`empty(get(b:, 'sy', {}))` is more inline with the code used elsewhere in Sy.

Reply:
Hmm, I guess not. Much of the changes in the commit that introduced this issue use `getbufvar()`.

Reply:
Can't agree with this. I copied this from `sy#toggle()` in `sy.vim`.
```shell
❯ rg -F 'get(b:'
❯
```
but
```shell
❯ rg -F 'getbufvar(bufnr('
autoload/sy.vim
86:  call call(empty(getbufvar(bufnr(''), 'sy')) ? 'sy#start' : 'sy#stop', [])
109:  return !empty(getbufvar(bufnr(''), 'sy'))

autoload/sy/repo.vim
260:  if empty(getbufvar(bufnr(''), 'sy'))

autoload/sy/util.vim
28:  if empty(getbufvar(bufnr(''), 'sy'))
100:  let sy = getbufvar(bufnr(''), 'sy')
```

Reply:
But I can change it anyway if you want to.

Reply:
Sorry, didn't refresh the page before sending the reply. Yeah, in most places `getbufvar` is used.